### ISSUE TITLE
Kolaboracja z innymi podobnymi projektami

### ISSUE THREAD

**Issue by niutech** [ProteGO-app/specs#73]
Zamiast od podstaw tworzyć własny projekt, nie lepiej połączyć szyki z innymi podobnymi projektami i współtworzyć jedną, ale skuteczną aplikację do śledzenia podejrzanych o koronawirusa?

- [WHO](https://worldhealthorganization.github.io/app/) pracuje nad oficjalną aplikacją, w kolejnych wersjach ma być śledzenie zarażonych koronawirusem,
- MIT już w marcu stworzył podobny projekt [Private Kit: Safe Paths](https://safepaths.mit.edu/) ([landing page](https://covidsafepaths.org/), [github](https://github.com/tripleblindmarket/private-kit)), który mocno skupia się na zachowaniu prywatności użytkowników - przechowywane dane są anonimowe,
- Stanford University pracuje nad [COVID Watch](https://www.covid-watch.org) ([github](https://github.com/covid19risk)) korzystającym z BLE,
- Ministerstwo Zdrowia Izraela wydało aplikację [HaMagen](https://govextra.gov.il/ministry-of-health/hamagen-app/download-en/) ([github](https://github.com/MohGovIL/hamagen-react-native), [opis](https://medium.com/proferosec-osm/hamagen-application-fighiting-the-corona-virus-4ecf55eb4f7c)),
- [CoEpi](https://www.coepi.org/) ([github](https://github.com/Co-Epi)) jest anonimowe i korzysta z Bluetooth,
- [Pandoa](https://pandoa.org/en/) ([github](https://github.com/wirewirewirewire/pandoa)) korzysta z GPS,
- [ITO STRICT](https://www.ito-app.org/) ([github](https://github.com/ito-org)) skupia się na zachowaniu prywatności korzystając z BLE,
- [CoronaTrace](https://www.coronatrace.org) ([github](https://github.com/Corona-Trace)) projekt podobny do Safe Paths,
- [PEPP-PT](https://www.pepp-pt.org/) (od jutra ma być na githubie) - pan-europejska inicjatywa śledzenia koronawirusa korzystając z Bluetooth z zachowaniem prywatności.

Zamiast wynajdywać koło na nowo i tworzyć n-tą aplikację do tego samego, nie lepiej dołączyć do obecnej inicjatywy w duchu open source?

**Comment by kierepka** [comment]
A tu chyba najpełniejsza lista: https://github.com/shankari/covid-19-tracing-projects

**Comment by niutech** [comment]
@kierepka Tak, ale nie chodzi mi o to, żeby publikować jak najpełniejszą listę podobnych projektów, tylko zaproponować kilka tych najbardziej znanych, do których można by się dołączyć, aby stworzyć jeszcze lepszą i bardziej popularną aplikację. Im większa społęczność, tm większe szanse na powodzenie projektów open source.

**Comment by Dr-Kownacki** [comment]
@niutech : myśmy się @kierepka "sami podłączyli" - wszystkie ręce na pokład :-))) #62  #63  #71 
Co możemy to podsylamy :-)))

**Comment by jakublipinski** [comment]
Wiem, że to żadna odpowiedź ale zaczęliśmy robić ProteGO zanim większość tych projektów była publiczna. Tego repo które wskazuje @kierepka też wtedy nie było.

@niutech który z tych projektów dogłębnie przeanalizowałeś i rekomendujesz zaadaptować w Polsce?

**Comment by niutech** [comment]
@jakublipinski Najbardziej w tej chwili perspektywiczne wydaje się być dołączenie do stawiającej nacisk na prywatność [TCN Coalition](https://tcn-coalition.org/) ([github](https://github.com/TCNCoalition)), w której są m. in.: [CoEpi](https://www.coepi.org/), [COVID Watch](https://www.covid-watch.org/), [ITO](https://www.ito-app.org/).

**Comment by wojtekszkutnik** [comment]
Prawdopodobnie https://techcrunch.com/2020/04/10/apple-and-google-are-launching-a-joint-covid-19-tracing-tool/ jest odpowiedzia na sporo problemow technicznych :)

**Comment by potiuk** [comment]
Prawdopodobnie pozamiatane (i bardzo dobrze). Zdecentralizowane, otwarte rozwiązanie działające z uprawnieniami pozwalającymi na nie usypianie. Inżynierowie Apple i Google patrzący sobie na ręce (i wszyscy patrzący im na ręce). Opt-in bez numeru telefonu.

**Comment by broukema** [comment]
ProteGO is no longer ;) missing from the en.Wikipedia article:

https://en.wikipedia.org/wiki/COVID-19_apps

There's a table listing (so far) five different specifications/protocol definitions: three are clearly free-licensed (DP-3T; BlueTrace/OpenTrace; TCN); one is "public" ([GAFAM](https://en.wikipedia.org/wiki/GAFAM)); and rather embarrassingly for trying to hold Europe together, the "pan-European" one is closed-source.

There's a long list of software projects. However, ProteGO is one out of _only four_ that is so far listed as being free-licensed, and one that is "compatible" with OpenTrace/BlueTrace. Most of the entries have no known licence (so far). (I looked randomly through a few of these. The quality varies: at [Aarogya Setu](https://www.aarogyasetu.in/2020/04/aarogya-setu-apk.html) I found the description _"Strictly, we are not hosting/sharing any modified (modded/cracked/patched) packages which can be dangerous for your device"_ rather unconvincing... :P

The other three free-licensed projects (eRouška, Hamagen, privatekit/safepaths) are under the MIT 3-clause licence, so that's forward compatible for integrating any useful parts into ProteGO under the GPLv3 or the GNU AGPLv3.

It would make sense to coordinate in some way with eRouška, Hamagen and privatekit/safepaths - e.g. crosspost on the git repositories, e.g. @augard, @simekadam.

**Comment by niutech** [comment]
@broukema AFAIK DP-3T is the [open protocol behind PEPP-PT](https://github.com/DP-3T/documents#the-relationship-between-dp-3t-and-pepp-pt), so it is not at all closed source.

**Comment by broukema** [comment]
@niutech I see your point; my comment was badly worded. I've edited my comment a bit - the link you give clarifies that DP-3T != PEPP-PT, and instead _"DP-3T participates in the loose umbrella of PEPP-PT"_. 

My earlier comment is also getting out of date compared to the en.Wikipedia table: all three of the CZ, DE and PL projects are free-licensed, which is not quite "pan-European", but it's heading in that direction. So far for FR we don't seem to have any info at all about STOPCOVID, only government announcements and [citizens' opposition](https://www.laquadrature.net/2020/04/14/nos-arguments-pour-rejeter-stopcovid/).
